### ISSUE TITLE
How to export nanodet onnx model with softmax and concat?

### ISSUE THREAD

**Issue by stuarteiffert** [linghu8812/tensorrt_inference#47]
Hi, 
I can't work out where the onnx model is edited during conversion as described in the comment made here https://github.com/RangiLyu/nanodet/issues/65

After running `python tools/export_onnx.py` my model still has 6 outputs instead of 1 concatenated output

![image](https://user-images.githubusercontent.com/16189580/107468307-d1550b00-6bbb-11eb-8843-1279e7144a6f.png)

**Comment by stuarteiffert** [comment]
Issue solved by ensuring that the edited nanodet is setup by running `python setup.py develop` in https://github.com/linghu8812/nanodet  before running export_onnx.py

**Comment by yueyihua** [comment]
@stuarteiffert @linghu8812 I have the same problem, I have run python setup.py develop in https://github.com/linghu8812/nanodet, but it's not useful.

**Comment by yueyihua** [comment]
The model output is like this:
![image](https://user-images.githubusercontent.com/10558846/115103237-f3378900-9f82-11eb-9a3b-98396584a7e7.png)

**Comment by linghu8812** [comment]
@yueyihua check these lines output
https://github.com/linghu8812/nanodet/blob/266321f33a39a2154fa5054f60fbb0acd983b906/nanodet/model/head/nanodet_head.py#L136-L141

**Comment by yueyihua** [comment]
@linghu8812 I used the newest pretrained model nanodet-m.ckpt from https://github.com/RangiLyu/nanodet.git, nanodet version is 0.3.0, and in your project https://github.com/linghu8812/nanodet.git, nanodet version is 0.1.0; if I use your code to setup, will cause an error: NotImplementedError.  I modify the code in https://github.com/RangiLyu/nanodet.git, it is not useful. What should I do, using your code to training?
